src/Silk.Shared/Configuration/SilkConfigurationOptions.cs

```diff
@@ -78,6 +78,8 @@ public class SilkPersistenceOptions
         public string Database { get; set; } = string.Empty;
         public string Username { get; set; } = "postgres";
         public string Password { get; set; } = string.Empty;
+
+        public override string ToString() => $"Host={Host}; Port={Port}; Database={Database}; Username={Username}; Password={Password};";
```

Comment:
Handy! This can be used in the `GetConnectionString` method in the parent class: `return Persistence.ToString();` 🙏